crates/pallet-domains/src/block_tree.rs

```diff
@@ -371,13 +375,13 @@ pub(crate) fn process_execution_receipt<T: Config>(
                 let bundle_digests = ExecutionInbox::<T>::get((
                     domain_id,
                     to_prune,
-                    execution_receipt.consensus_block_number,
+                    execution_receipt.consensus_block_number(),
```

Comment:
We repeatedly call `execution_receipt.consensus_block_number()` in this function, would it be better to cache this value in a variable once rather than making repeated method calls? I've notice this is how all of the `execution_receipt` fields are being handled, so maybe there is a reason for it.